raffia/src/parser/builder.rs

```diff
@@ -7,7 +7,7 @@ use crate::{
 /// Parser builder is for building a parser while allowing us
 /// to control advanced behaviors.
 ///
-/// Unlike [`Parser`], syntax isn't required when creating a parser builder,
+/// Unlike [Parser], syntax isn't required when creating a parser builder,
```

Comment:
Please don't remove the link.

Reply:
![image](https://user-images.githubusercontent.com/17974631/192130716-533e8455-4502-4acf-991d-68b0119f808c.png)
code block not necessary if you only want the link,

Reply:
But I have reverted it.